JetStreamDriver.js

```diff
@@ -781,16 +826,36 @@ class Benchmark {
         return geomeanScore(subScores);
     }
 
+    get totalTime() {
```

Comment:
More high-level: Instead of having individual fields for each score and time, including these getters, how about we simply have the `{sub,all}{Scores,Times}` objects with simply the keys defining what's available?

Reply:
yeah, I have this still pending somwhere... just wanted to get this out with fewer changes. 
I might do this as a followup.